portfolio/style.css

```diff
@@ -0,0 +1,515 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Quicksand;
+  src: url(/MultiPage-Clone/fonts/Quicksand/Quicksand-VariableFont_wght.ttf);
+  font-family: poppins;
+  src: url(MultiPage-Clone/fonts/poppins);
+}
+
+/* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --clr-Light-green: #436850;
+  --clr-lite-gray: #d3d3d3;
+  --clr-Black: #0b0b0b;
+  --clr-White: #fff;
+  --clr-dark-green: #12372a;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 130px;
+  background: #12372a;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 999;
+}
+
+.nav-links {
+  list-style: none;
+  display: flex;
+  text-transform: uppercase;
+}
+
+ul {
+  margin-right: 2rem;
+}
+
+.nav-links li {
+  padding: 0 20px;
+}
+
+.nav-links a {
+  color: var(--clr-White);
+  text-decoration: none;
+  font-size: 1.3rem;
+  font-family: Quicksand;
+  font-weight: bold;
+}
+
+.nav-links a:hover {
+  color: var(--clr-lite-gray);
+}
+
+.container {
+  margin-top: 230px;
+  font-size: 2rem;
+  display: inline-block;
+  overflow: hidden;
+  position: relative;
+  width: 400px;
+  height: 250px;
+  left: 40%;
+}
+
+.container span {
+  font-size: 5rem;
+  font-family: poppins;
+}
+
+.text {
+  position: absolute;
+  width: 100%;
+  text-align: center;
+  opacity: 0;
+  transform: translateY(100%);
+  animation: slide 9s infinite;
+}
+
+.text:nth-child(1) {
+  animation-delay: 0s;
+}
+
+.text:nth-child(2) {
+  animation-delay: 3s;
+}
+
+.text:nth-child(3) {
+  animation-delay: 6s;
+}
+
+@keyframes slide {
+  0%,
+  20% {
+    opacity: 0;
+    transform: translateY(100%);
+  }
+
+  25%,
+  45% {
+    opacity: 1;
+    transform: translateY(0);
+  }
+
+  50%,
+  100% {
+    opacity: 0;
+    transform: translateY(-100%);
+  }
+}
+
+@media (min-width: 300px) and (max-width: 800px) {
+  nav {
+    height: 100px;
+    justify-content: center;
+    align-items: center;
+  }
+  .nav-links a {
+    font-size: 0.9rem;
+  }
+  .nav-links li {
+    padding: 10px;
+  }
+  .container {
+    width: 90%;
+    height: 100px;
+    left: 5%;
+    margin-top: 150px;
+  }
+
+  .container span {
+    font-size: 2rem;
+  }
+
+  .text {
+    font-size: 1.5rem;
+  }
+
+  .text:nth-child(1) {
```

Comment:
A bit of duplication. The same delays are common for other page sizes as well.
Please check what else is duplicated

Reply:
ok i will check this